lib/screens/home/asset/asset_chart_recent_transactions.dart

```diff
@@ -15,7 +14,7 @@ class AssetChartRecentTransactions extends StatelessWidget {
   final Asset asset;
   @override
   Widget build(BuildContext context) {
-    final bloc = Provider.of<TransactionsBloc>(context);
+    final bloc = Provider.of<TransactionsBloc>(context, listen: false);
```

Comment:
Probably want `listen: true` so that this widget rebuilds if the `TransactionsBloc` gets a new instance